client/src/Categories/CategoriesSources.js

```diff
@@ -0,0 +1,40 @@
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+
+import './CategoriesList.scss';
+import { useEffect, useState } from 'react';
+import Api from '../Api';
+
+function CategoriesList() {
+  const { slug } = useParams();
+  const [category, setCategory] = useState([]);
```

Comment:
category is a single object, so its initial value should be null or `{}`, not an array, I think...

Reply:
I added `{}` to line 10 inside `useState`